OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -125,7 +125,7 @@ IEnumerable<EditorActorOption> IEditorActorOptions.ActorOptions(ActorInfo ai, Wo
 		}
 	}
 
-	public class Mobile : ConditionalTrait<MobileInfo>, INotifyCreated, IIssueOrder, IResolveOrder, IOrderVoice, IPositionable, IMove,
+	public class Mobile : PausableConditionalTrait<MobileInfo>, INotifyCreated, IIssueOrder, IResolveOrder, IOrderVoice, IPositionable, IMove,
```

Comment:
Not your fault, but since I noticed it now would you mind removing the redundant `INotifyCreated` here?

Reply:
Done.